### PR TITLE
Release of the Go Version

### PR DESCRIPTION
**Pull request by vimalsheoran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+# Introduction
+
+This library provides easy to use interface of the Bolt Cloud APIs. The functions are similar in syntax to standard Arduino or Rasberry Pi functions.
+
+# Installation
```
Comment (rsbuilds) [inline README.md:7]: I think this is a good way for installing the go package. 

Installing
```

go get -u github.com/Inventrom/bolt-api-go

Example
Getting all groups
import (
	"fmt"

	"github.com/Inventrom/bolt-api-go"
)
```

**Comment by rsbuilds** [comment]
@vimalsheoran Function format is incorrect. We should follow the same naming convention for all the public function name. 
For example 
Current - 
```
func main(){
	response := bolt.DigitalWrite("2","HIGH")
	fmt.Println(response)
}
```
It should be  like this snippet . 
```
func main(){
	response := bolt.digitalWrite("2","HIGH")
	fmt.Println(response)
}
```

File: README.md (hunk 1 above)
Comment (vimalsheoran) [reply README.md:7]: I will implement that.

**Comment by vimalsheoran** [comment]
@quelist In Go, functions that are to be exported as package functions have to be camel cased and need to have their first letter as a capital letter, e.g fmt.Println().

**Review by rsbuilds — APPROVED** [review]

**Review by sashoeb94 — COMMENTED** [review]
Reviewed Code and tested the code as well as the test cases.

**Pull request merged**